bin/travis-before-install.sh

```diff
@@ -15,6 +15,18 @@ if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
    unzip ninja-linux.zip
    sudo cp ninja /usr/bin
 
+   # Install openssl 1.1.0i
+   ldconfig -p | grep libssl
+   readlink /usr/lib/x86_64-linux-gnu/libssl.so
+   wget --quiet https://www.openssl.org/source/openssl-1.1.0i.tar.gz
+   tar xfz openssl-1.1.0i.tar.gz
+   cd openssl-1.1.0i
+   ./config --prefix=/usr --libdir=lib/x86_64-linux-gnu
```

Comment:
Default is installing in `/usr/local` but we want to have a global install, since we can't modify node bindings .gyp file from here.